tempesta_fw/http_parser.c

```diff
@@ -1592,8 +1601,11 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len)
 
 	/* Request headers are fully read. */
 	__FSM_STATE(Req_HdrDone) {
-		if (c == '\n')
+		if (c == '\n') {
+			TFW_LOG("CRLF end\n");
+			__field_finish(msg, &req->crlf, data, p + 1);
```

Comment:
The fact that we don't set the length of `req->crlf` string until it's complete is somewhat cumbersome to remember and rely on. For that reason I believe that using `TFW_STR_EMPTY()` on `req->crlf` is cumbersome as well. Now that we set `req->crlf` correctly, I believe the correct way of checking for it in `Frang` is checking for `TFW_STR_COMPLETE` flag. What do you think?